src/audio/module_adapter/module/generic.c

```diff
@@ -336,6 +336,9 @@ int module_reset(struct processing_module *mod)
 	if (md->state < MODULE_IDLE)
```

Comment:
minor: some typoes in commit, "s/posiibility/possibility", "s/sollution/solution/",